sources/bloodhound/src/bin/kubernetes-checks/checks.rs

```diff
@@ -0,0 +1,714 @@
+                } else {
+                    result.status = CheckStatus::PASS;
+                }
+            } else {
+                result.error = "unable to parse kubelet config".to_string()
+            }
+        } else {
+            result.error = format!("unable to read '{}'", KUBELET_CONF_FILE);
+        }
+
+        result
+    }
+
+    fn metadata(&self) -> CheckerMetadata {
+        CheckerMetadata {
+            title: "Verify that the RotateKubeletServerCertificate argument is set to true"
+                .to_string(),
+            id: "4.2.12".to_string(),
+            level: 1,
+            name: "k8s04021200".to_string(),
+            mode: Mode::Automatic,
+        }
+    }
+}
+
+// =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<=
+
+pub struct K8S04021300Checker {}
+
+impl Checker for K8S04021300Checker {
+    fn execute(&self) -> CheckerResult {
+        let allowed_suites = vec![
+            "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+            "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
+            "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
+            "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
+            "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
+            "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
+            "TLS_RSA_WITH_AES_256_GCM_SHA384",
+            "TLS_RSA_WITH_AES_128_GCM_SHA256",
+        ];
+
+        #[derive(Deserialize)]
+        struct KubeletConfig {
+            #[serde(rename = "tlsCipherSuites")]
+            tls_cipher_suites: Vec<String>,
+        }
+
+        let mut result = CheckerResult::default();
+
+        if let Ok(kubelet_file) = File::open(KUBELET_CONF_FILE) {
+            if let Ok(config) = serde_yaml::from_reader::<_, KubeletConfig>(kubelet_file) {
+                let mut allowed = true;
+                for suite in config.tls_cipher_suites {
+                    if !allowed_suites.iter().any(|&i| i == suite) {
+                        allowed = false;
+                        result.error = format!("'{}' not an allowed cipher suite", suite);
+                        break;
+                    }
+                }
+
+                if !allowed {
+                    result.status = CheckStatus::FAIL;
+                } else {
+                    result.status = CheckStatus::PASS;
+                }
+            } else {
+                result.error = "unable to parse kubelet config".to_string()
+            }
+        } else {
+            result.error = format!("unable to read '{}'", KUBELET_CONF_FILE);
+        }
+
+        result
+    }
+
+    fn metadata(&self) -> CheckerMetadata {
+        CheckerMetadata {
+            title: "Ensure that the Kubelet only makes use of Strong Cryptographic Ciphers"
+                .to_string(),
+            id: "4.2.13".to_string(),
+            level: 1,
+            name: "k8s04021300".to_string(),
+            mode: Mode::Automatic,
+        }
+    }
```

Comment:
We are missing the current `4.2.13` check - "Ensure that a limit is set on pod PIDs (Manual)".